src/main/webapp/dataverse_template.xhtml

```diff
@@ -51,6 +51,9 @@
                 <o:resourceInclude path="/CustomizationFilesServlet?customFileType=style" />
             </style>
         </ui:fragment>
+        <ui:fragment rendered="#{DatasetPage.getCVocConf().size() > 0}">
+            <script src="#{resource['js/jquery-ui.js']}"></script>
```

Comment:
Will adding jQuery UI change the look and feel of Dataverse?

Reply:
removed - that was a left-over from the original DANS submission. I ended up using the select2 library instead. In any case, given that these libraries shouldn't be getting used anywhere else in Dataverse, their inclusion shouldn't affect the UI outside the fields being mapped via the external vocab support.